bundle/Resources/translations/messages.fr.yml

```diff
@@ -34,6 +34,8 @@ netgen_admin_ui.menu.cache: 'Cache'
 netgen_admin_ui.menu.clear_cache: 'Vider le cache'
 netgen_admin_ui.menu.bookmarks: 'Les signets'
 
+netgen_admin_ui.menu.information_collection: 'Collected information'
```

Comment:
No need to have untranslated strings in non english language files. This defeats the purpose when Symfony debug toolbar has the option of reporting untranslated strings.